Formula/faunus.rb

```diff
@@ -0,0 +1,66 @@
+class Faunus < Formula
+  include Language::Python::Virtualenv
+
+  desc "Framework for Metropolis Monte Carlo Simulation of Molecular Systems"
+  homepage "https://mlund.github.io/faunus"
+  url "https://github.com/mlund/faunus/archive/522213d.tar.gz"
+  version "2.5.0"
+  sha256 "d7b3d6bfddafa808fd21108c916583c850719d736a2f344aac3ab6088d2d72cc"
+  license "MIT"
+  revision 1
```

Comment:
This isn't needed, 0 is the default (and that should be used)

Reply:
`revision` removed